public/app/plugins/panel/graph/module.ts

```diff
@@ -311,11 +311,10 @@ class GraphCtrl extends MetricsPanelCtrl {
   }
 
   exportCsv() {
-    var scope = this.$scope.$new();
+    var scope = this.$scope.$new(true);
     scope.seriesList = this.seriesList;
-
     this.publishAppEvent('show-modal', {
-      src: 'public/app/features/dashboard/partials/exportCsvModal.html',
+      templateHtml: '<export-data-modal data="seriesList"></export-data-modal>',
```

Comment:
I renamed it to export-data-modal because the prefix "data-" is stripped away during the directive name normalization: See: https://docs.angularjs.org/guide/directive#normalization

Same for the attribute "data". The attribute "data-list" would resolve to "list" in the directive..